docs/velocity/admin/getting-started/faq.mdx

```diff
@@ -17,7 +17,7 @@ Velocity 3.3.x requires Java <Property name="VELOCITY_JAVA_MIN" /> or above.
 A good source for finding plugins compatible with Velocity would be our plugin repository
 [Hangar](https://hangar.papermc.io/?page=0&platform=VELOCITY).
 
-## Does Velocity support plugins developed for BungeeCord or Waterfall?
+## Does Velocity support plugins developed for BungeeCord?
```

Comment:
Since this is a FAQ I'd say it's justified to keep Waterfall here

Reply:
Fixed